From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: mlaurencin <mlaurencin@electronjs.org>
Date: Fri, 6 Aug 2021 15:29:48 -0700
Subject: disable use_lld for macOS

This patch disables use_lld on macOS, in order to prevent a linking bug
that occurs when building for arm64.

diff --git a/build/config/compiler/compiler.gni b/build/config/compiler/compiler.gni
index 4a714ab0a5cdafded76bba553205f94599c27a30..d2723dd224a8afd2b6dd99ec11e9e71b7f76fe4a 100644
--- a/build/config/compiler/compiler.gni
+++ b/build/config/compiler/compiler.gni
@@ -201,7 +201,7 @@ declare_args() {
   # In late bring-up on macOS (see docs/mac_lld.md), and not functional at all for
   # iOS. The default linker everywhere else.
   use_lld =
-      is_clang && (!is_apple || (target_os == "mac" && chrome_pgo_phase != 1))
+      is_clang && !is_apple
 }
 
 declare_args() {
